namespaces/live.cloud-platform.service.justice.gov.uk/whereabouts-api-prod/resources/rds.tf

```diff
@@ -2,20 +2,22 @@
 variable "vpc_name" {
 }
 
-
 module "dps_rds" {
-  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.16.16"
-  vpc_name               = var.vpc_name
-  team_name              = var.team_name
-  business-unit          = var.business-unit
-  application            = var.application
-  is-production          = var.is-production
-  namespace              = var.namespace
-  environment-name       = var.environment-name
-  infrastructure-support = var.infrastructure-support
-  db_password_rotated_date    = "15-02-2023"
```

Comment:
With https://github.com/ministryofjustice/cloud-platform-environments/pull/10626 we also dropped `db_password_rotated_date`. Unexpectedly this caused the password to be rotated _again_ which wasn't ideal, but we had planned downtime and the API processes had already been brought down so no major impact. And they then picked up the new passwords when we brought them back

Doing this here because didn't want to leave config which someone may remove at a later date without understanding the impact